h5d.go

```diff
@@ -177,11 +177,71 @@ func (s *Dataset) OpenAttribute(name string) (*Attribute, error) {
 	return openAttribute(s.id, name)
 }
 
+//H5_DLL hid_t H5Dget_space(hid_t dset_id);
+func (s *Dataset) H5Dget_space() (*Dataspace, error) {
+	dspace_id := (C.H5Dget_space(s.id))
```

Comment:
It looks like there are residual conversion parens. Can you clean these up please.